Documentation/FileFormats.md

```diff
@@ -0,0 +1,58 @@
+# Serenity Formats
+
+This document summarizes all non-standard file formats, data formats, and mime-types used by Serenity.
+
+## glyph/x-fonteditor (Clipboard-only)
+
+Requires the metadata-fields, `char` (usually encoded as a string of length 1: the codepoint as a character),
+`width`, and `height` (encoded as a decimal number).
+
+The data is encoded in width times height many bytes, either 0 (clear) or 1 (set).
+
+## image/x-serenityos (Clipboard-only)
+
+Requires the metadata-fields `width`, `height`, `scale`, `format` (see `Gfx::BitmapFormat`), and `pitch`, encoded as decimal strings.
+
+The data is encoded according to `Gfx::determine_storage_format(BitmapFormat)`, so either as
+BGRx8888, BGRA8888, RGBA8888, or 8-bit pallette index. Note that the pallette is not transferred.
+
+## text/uri-list (Clipboard-only)
+
+Newline-delimited set of URIs.
```

Comment:
would also appreciate an example here, e.g.

```suggestion
Newline-delimited set of URIs.

Example:

```
http://serenityos.org/
https://example.com/foo/bar
```
```

(right?)

Reply:
No, but I'll replace it with a better example, and a list of applications that support it.

Reply:
oh, URIs, not URLs. Yeah, I remember what this is about now. Proper example would be nice, thanks :)